### PR TITLE
Reminder feuture

### PR DESCRIPTION
**Pull request by Sikkis**
A ctf command to remind  the members with a message before the start of the CTF

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ _NOTE: the following ctf specific commands will only be accepted under the categ
 
 - `!ctf showcreds` Reveals shared credentials that have been set for the CTF
 
+- `ctf reminder` Sets a reminder to alert the team members before the CTF starts.
```
Comment (apogiatzis) [inline README.md]: !ctf reminder. "!" missing

File: bot/core.py [hunk 2]
```diff
@@ -143,6 +149,25 @@ def run():
...
+@tasks.loop(seconds=1800)
+async def reminder():
+    guild = bot.guilds[0]
+    categories = guild.by_category()
+    ctfs = [c for c in guild.categories if c.name != 'Text Channels' and c.name != 'Voice Channels' ]
+    for ctf in ctfs:
+        try:
+            ctf_doc = CTF.objects.get({"name": ctf.name})
+            if ctf_doc.reminder:
+                for c in categories:
```
Comment (apogiatzis) [inline bot/core.py]: Nomizw dame mporeis na fieis to deutero loop jai na xrisimopiisis apla to `ctf` var epd etsi gireukis mesta categories 2 fores

File: bot/core.py [hunk 3]
```diff
@@ -31,6 +35,8 @@
     logger.info(('<' + bot.user.name) + ' Online>')
     logger.info(discord.__version__)
     await bot.change_presence(activity=discord.Game(name='with your mind! Use !help'))
+    #bot.bg_task = bot.loop.create_task(my_background_task(bot))
```
Comment (apogiatzis) [inline bot/core.py]: Last thing, eshi dame ena comment j allona sto db_models an enei fie ta na en pio katharo

File: bot/core.py (hunk 2 above)
Comment (Sikkis) [reply bot/core.py]: en mporeis  giati to ctf en categories eno to allo eshei mesa ta channels  p exei  to  to ka8e category
.

**Comment by Sikkis** [comment]
Done

File: bot/core.py (hunk 2 above)
Comment (apogiatzis) [reply bot/core.py]: an kamis kati etsi gia na fiei jino to loop? `channel = discord.utils.get(ctf.channels, name='general')`

Comment (Sikkis) [reply bot/core.py]: ne ma pio general en na ine? egw gireukw  to general tou specific CTF en kamei specify pio pou oula en na ferei dame

Comment (Sikkis) [reply bot/core.py]: resolved

**Pull request merged**